Clarify package manager instructions (#17485)
dotnet/docs: docs/core/install/linux-package-manager-debian10.md

description: Use a package manager to install .NET Core SDK and runtime on Debian 10.
author: thraka
ms.author: adegeo
ms.date: 03/17/2020
---

# Debian 10 Package Manager - Install .NET Core

[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]

> This article describes how to use a package manager to install .NET Core on Debian 10. Install the SDK (includes the runtime) if you want to make apps with .NET Core. Or, install the runtime if you only need to run apps. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.

Reviewer: ```suggestion
This article describes how to use a package manager to install .NET Core on Debian 10. If you want to develop .NET Core apps, install the SDK (which includes the runtime). Or, if you only need to run apps, install the runtime. If you're installing the runtime, consider installing the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), which also includes the .NET Core runtime.
```
